rust/src/ffi/steppable_evmc_vm.rs

```diff
@@ -0,0 +1,181 @@
+use std::{ffi::c_void, panic, slice};
+
+use ::evmc_vm::{
+    ffi::{
+        evmc_bytes32, evmc_capabilities, evmc_host_interface, evmc_message, evmc_revision,
+        evmc_step_result, evmc_step_status_code, evmc_vm as evmc_vm_t, evmc_vm_steppable,
+    },
+    ExecutionContext, ExecutionMessage, StatusCode, StepResult, StepStatusCode,
+    SteppableEvmcContainer, SteppableEvmcVm,
+};
+
+use crate::{
+    evmc::EvmRs,
+    ffi::evmc_vm::{self, EVMC_CAPABILITY},
+};
+
+#[no_mangle]
+extern "C" fn evmc_create_steppable_evmrs() -> *const evmc_vm_steppable {
+    let new_instance = evmc_vm_steppable {
+        vm: evmc_vm::evmc_create_evmrs() as *mut evmc_vm_t,
+        step_n: Some(__evmc_step_n),
+        destroy: Some(__evmc_steppable_destroy),
+    };
+    let container = SteppableEvmcContainer::<EvmRs>::new(new_instance);
+
+    // Release ownership to EVMC.
+    unsafe { SteppableEvmcContainer::into_ffi_pointer(container) }
+}
+
+extern "C" fn __evmc_steppable_destroy(instance: *mut evmc_vm_steppable) {
+    if instance.is_null() {
+        // This is an irrecoverable error that violates the EVMC spec.
+        std::process::abort();
```

Comment:
Not sure about this neither, I see where it comes from, but I guess that at this point it could be ignored (just don't drop it if null)

Reply:
Here I would not have an issue with ignoring it. Will change this.